metadata/contributors.json

```diff
@@ -3142,7 +3142,7 @@
    },
    {
       "emails" : [
-         "Hironori.Fujii@sony.com",
```

Comment:
I think that we prefer to keep the old address, so that it's easy to analyze old commit messages. Would you be OK with keeping it below the current one?